### PR TITLE
Bug #154594 fix : Adding custom layout(12X12)

### PR DESCRIPTION
**Pull request by ChavanAnkitaMahesh**

### CODE REVIEW

File: src/components/com_tjucm/site/views/itemform/tmpl/grid.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<metadata>
+	<layout title="COM_TJUCM_TITLE_FORM_VIEW_ITEMFORM" option="One line one input">
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/site/views/itemform/tmpl/grid.xml]: @ChavanAnkitaMahesh `One line one input` this must be a language constant
Also, check the value of the option for the default layout XML

Update `One line one input` => `One input in a row`

File: src/components/com_tjucm/site/views/itemform/tmpl/grid.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @version    SVN: <svn_id>
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/site/views/itemform/tmpl/grid.php]: @ChavanAnkitaMahesh use the updated comment style for new files
[code]
/**
 * @package     TJ-UCM
 * @subpackage  com_tjucm
 *
 * @author      Techjoomla <extensions@techjoomla.com>
 * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
 * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
 */
[/code]

File: src/components/com_tjucm/site/views/itemform/tmpl/default.php [hunk 3]
```diff
@@ -41,6 +41,7 @@
 $baseUrl                   = $jinput->input->server->get('REQUEST_URI', '', 'STRING');
 $calledFrom                = (strpos($baseUrl, 'administrator')) ? 'backend' : 'frontend';
 $layout                    = ($calledFrom == 'frontend') ? 'default' : 'edit';
+$dyanamic_layout=$this->setLayout($typeParams->layout);
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/site/views/itemform/tmpl/default.php]: @ChavanAnkitaMahesh use camel case style for variable names in PHP
`$dyanamic_layout` => `$layoutToUse`

File: src/components/com_tjucm/site/views/itemform/tmpl/grid.xml (hunk 1 above)
Comment (ChavanAnkitaMahesh) [reply src/components/com_tjucm/site/views/itemform/tmpl/grid.xml]: resolved

File: src/components/com_tjucm/site/views/itemform/tmpl/default.php (hunk 3 above)
Comment (ChavanAnkitaMahesh) [reply src/components/com_tjucm/site/views/itemform/tmpl/default.php]: resolved

**Pull request merged**